dist/carousel/ds4/carousel.css

```diff
@@ -57,6 +57,26 @@
 .carousel__playback .icon--play {
   padding-left: 1px;
 }
+.carousel__container--image-treatment-height .carousel__list img,
```

Comment:
I think these dist files are out if sync with the src files now, right?

Reply:
yeah I think so. I guess I need to re-run the build